content/influxdb/v1.4/troubleshooting/query_management.md

```diff
@@ -141,6 +141,8 @@ the following error:
 ```
 ERR: max number of points reached
 ```
+The point count will be checked every second, so queries will not be aborted immediately upon hitting the limit.
```

Comment:
Thoughts on streamlining and weaving this line into text above? Something like:

The maximum number of points a SELECT statement can process (default 0 allows unlimited points). 

InfluxDB checks the point count every second (so queries aren’t immediately aborted). If your query exceeds `max-select-point`, InfluxDB stops the query and outputs the following error: `ERR: max number of points reached`

Reply:
@kelseiv updated and rebased